mystery_o_matic/output/latex/__init__.py

```
@@ -103,7 +108,15 @@ def produce_tex_output(static_dir, out_dir, languages, mystery, weapons, weapon_
             clue = replace_opening_quotes(clue)
             additional_clues.append(clue)
 
-        additional_clues_enumeration = get_enumeration_list(additional_clues)
+        additional_clues_enumeration = get_bullet_list(additional_clues, customItem="\\ding{43}")
+
+        additional_clues_with_lies = []
+        for i, clue in enumerate(mystery.additional_clues_with_lies):
```

Ruff

Loop control variable `i` not used within loop body

(B007)